### DISCUSSION TITLE
Whats gonna happen when destination drive is full / file copy error

### DISCUSSION THREAD

**Discussion by vvavepacket** [madMAx43v3r/chia-plotter#523] [Q&A]
We all know file copy now to destination folder is async..

Whats going to happen to that async process once the destination folder becomes full? Will it crash or kill the async process and never retried again?

**Comment by cyperbg** [comment]
It will try and fail every x number of minutes.

**Reply by andymarden** [reply]
That's better than the official plotter which seems to say "oh well never mind", delete your original plot file and leave a half-finished  file in the dest dir (ffs)

**Reply by carlfarrington** [reply]
I found that the original plotter did the same. You just kill the process, remove the .tmp files, and the .plot file is there for you to copy manually.

The difference is that the original plotter tells yous 'disk full, will retry in 5 minutes', while Max's plotter says 'fwrite() failed'.
